src/scss/constants.scss

```diff
@@ -63,6 +63,8 @@ $playa-avatar-size: 48px;
 $playa-venue-live: rgb(125, 223, 194);
 
 $margin-right--navbar-links: 8px;
+$edit-topic-button-diameter: 0px 5px;
+$edit-table-footer-padding: 10px 0 0 0;
```

Comment:
IMO these should be in the file itself instead of the global one. This will be flooded with concrete and component specific styles. Which will have the same values but different names. Maybe we can add padding constants here with common/generic names that can be reused.

Reply:
Generic named padding constants with a mind to moving towards standardised paddings if that makes sense sounds like a good idea.

As far as component specific variables, they can probably live in their own files. The only times it makes sense to bring up to the constants file is when multiple components need to depend on that var (eg. Navbar/footer heights come to mind here as an example)

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/c222264f9875e83b03a9a64b49c9ae1d57e9401c